docs/design/parameters.md

```diff
@@ -0,0 +1,81 @@
+# Gravity parameters
```

Comment:
This is already within the spec, why was it added here?

Reply:
there is currently no parameters spec in the repo. Nor an open PR with any specs. Where would I go to find it?

Reply:
https://github.com/cosmos/gravity-bridge/blob/main/module/x/gravity/spec/07_params.md

this was created a while ago. It lists the params but does not explain them.